test/unit/manipulation.js

```diff
@@ -3099,3 +3099,55 @@ testIframe(
 		} );
 	}
 );
+
+// Custom event handler test case
+
+QUnit.test( "should handle custom '_se-custom-destroy' event correctly", function( assert ) {
+
+	// Define the expected number of assertions
```

Comment:
Avoid comments that don't bring any additional value. If it's obvious from the code what it does, a comment shouldn't be added.

Reply:
Done